cli/mender-artifact/debugfs.go

```diff
@@ -74,12 +74,16 @@ func debugfsCopyFile(file, image string) (ret string, err error) {
 }
 
 func debugfsMakeDir(imageFile, image string) (err error) {
+	_, err = debugfsExecuteCommand(fmt.Sprintf("stat %s", imageFile), image)
```

Comment:
Should this also go after the `TrimRight`?

Reply:
`debugfs stat` doesn't mind the trailing `/`, but `mkdir` does. I've clarified the comment to say it is needed for the `mkdir` command